packages/wallet-adapter-react/src/useWallet.tsx

```diff
@@ -36,32 +39,27 @@ export interface WalletContextState {
   disconnect(): void;
   wallet: WalletInfo | null;
   wallets: ReadonlyArray<Wallet>;
-  signAndSubmitTransaction<T extends Types.TransactionPayload>(
-    transaction: T,
-    options?: TransactionOptions
-  ): Promise<any>;
-  signAndSubmitBCSTransaction<T extends TxnBuilderTypes.TransactionPayload>(
-    transaction: T,
-    options?: TransactionOptions
```

Comment:
Okay, so this will no longer work.  Good to know for future dapps to be put in a changelog